Microsoft.Research/Clousot.Cache/Models/Mapping/SuggestionMap.cs

```diff
@@ -30,7 +30,7 @@ public SuggestionMap()
             // Properties
             this.Property(t => t.Kind)
                 .IsRequired()
-                .HasMaxLength(100);
+                .HasMaxLength(400);
```

Comment:
I'd like to see the constant added to (and used in) [`MethodMap`](https://github.com/Microsoft/CodeContracts/blob/c71ef8f1dee388ae3559c2f92969849c977c2d35/Microsoft.Research/Clousot.Cache/Models/Mapping/MethodMap.cs#L37), and then referenced from this location.